composer.json

```diff
@@ -66,7 +66,8 @@
         "symfony/phpunit-bridge": "2.8.10",
         "symfony/monolog-bundle": "2.8.10",
         "jackalope/jackalope": "1.2.8 || 1.3.0 || 1.3.1",
-        "phpcr/phpcr-utils": "1.2.0 - 1.2.10 || 1.3.0 - 1.3.2"
+        "phpcr/phpcr-utils": "1.2.0 - 1.2.10 || 1.3.0 - 1.3.2",
+        "twig/twig": ">=2.7"
```

Comment:
in 2.7 `\Twig_Runtime_Error` was deprecated and therfor the tests fail because of a deprecation notice. /cc @danrot @alexander-schranz what do you think - conflict or weak deprecation handler?

Reply:
set deprecation helper to weak